@codex please fix

```
2026-04-12T07:09:13.4739685Z Current runner version: '2.333.1'
2026-04-12T07:09:13.4763327Z ##[group]Runner Image Provisioner
2026-04-12T07:09:13.4764429Z Hosted Compute Agent
2026-04-12T07:09:13.4764902Z Version: 20260213.493
2026-04-12T07:09:13.4765502Z Commit: 5c115507f6dd24b8de37d8bbe0bb4509d0cc0fa3
2026-04-12T07:09:13.4766115Z Build Date: 2026-02-13T00:28:41Z
2026-04-12T07:09:13.4766661Z Worker ID: {bf218e3a-0ee7-47f5-bb17-255063c1e1de}
2026-04-12T07:09:13.4767292Z Azure Region: northcentralus
2026-04-12T07:09:13.4767780Z ##[endgroup]
2026-04-12T07:09:13.4769448Z ##[group]Operating System
2026-04-12T07:09:13.4769997Z Ubuntu
2026-04-12T07:09:13.4770416Z 24.04.4
2026-04-12T07:09:13.4770820Z LTS
2026-04-12T07:09:13.4771267Z ##[endgroup]
2026-04-12T07:09:13.4771714Z ##[group]Runner Image
2026-04-12T07:09:13.4772152Z Image: ubuntu-24.04
2026-04-12T07:09:13.4772626Z Version: 20260406.80.1
2026-04-12T07:09:13.4773661Z Included Software: https://github.com/actions/runner-images/blob/ubuntu24/20260406.80/images/ubuntu/Ubuntu2404-Readme.md
2026-04-12T07:09:13.4774895Z Image Release: https://github.com/actions/runner-images/releases/tag/ubuntu24%2F20260406.80
2026-04-12T07:09:13.4775682Z ##[endgroup]
2026-04-12T07:09:13.4776651Z ##[group]GITHUB_TOKEN Permissions
2026-04-12T07:09:13.4778577Z Contents: read
2026-04-12T07:09:13.4779070Z Metadata: read
2026-04-12T07:09:13.4779496Z PullRequests: read
2026-04-12T07:09:13.4779983Z ##[endgroup]
2026-04-12T07:09:13.4781993Z Secret source: Actions
2026-04-12T07:09:13.4782613Z Prepare workflow directory
2026-04-12T07:09:13.5264507Z Prepare all required actions
2026-04-12T07:09:13.5301599Z Getting action download info
2026-04-12T07:09:13.9206526Z Download action repository 'dorny/paths-filter@de90cc6fb38fc0963ad72b210f1f284cd68cea36' (SHA:de90cc6fb38fc0963ad72b210f1f284cd68cea36)
2026-04-12T07:09:14.3575690Z Download action repository 'actions/checkout@v5' (SHA:93cb6efe18208431cddfb8368fd83d5badbf9bfd)
2026-04-12T07:09:14.3834020Z Download action repository 'docker/setup-qemu-action@v3' (SHA:c7c53464625b32c7a7e944ae62b3e17d2b600130)
2026-04-12T07:09:14.9375666Z Download action repository 'docker/setup-buildx-action@v3' (SHA:8d2750c68a42422c14e847fe6c8ac0403b4cbd6f)
2026-04-12T07:09:15.5267480Z Complete job name: oci-parity-smoke
2026-04-12T07:09:15.5990371Z ##[group]Run dorny/paths-filter@de90cc6fb38fc0963ad72b210f1f284cd68cea36
2026-04-12T07:09:15.5991955Z with:
2026-04-12T07:09:15.5994219Z   filters: oci:
  - '.github/workflows/pi-image.yml'
  - '**/Dockerfile'
  - '**/Dockerfile.*'
  - '**/package.json'
  - '**/package-lock.json'
  - '**/pnpm-lock.yaml'
  - 'deploy/**'
  - 'infra/**'
  - 'platform/**'

2026-04-12T07:09:15.5996938Z   token: ***
2026-04-12T07:09:15.5997595Z   list-files: none
2026-04-12T07:09:15.5998441Z   initial-fetch-depth: 100
2026-04-12T07:09:15.5999412Z env:
2026-04-12T07:09:15.6000101Z   FORCE_JAVASCRIPT_ACTIONS_TO_NODE24: true
2026-04-12T07:09:15.6001010Z   CLONE_DSPACE: 
2026-04-12T07:09:15.6001688Z ##[endgroup]
2026-04-12T07:09:15.6813606Z ##[group]Fetching list of changed files for PR#2162 from Github API
2026-04-12T07:09:15.6815288Z Invoking listFiles(pull_number: 2162, per_page: 100)
2026-04-12T07:09:15.8941732Z Received 2 items
2026-04-12T07:09:15.8944025Z [modified] .github/workflows/pi-image.yml
2026-04-12T07:09:15.8948599Z [modified] tests/test_pi_image_tooling.py
2026-04-12T07:09:15.8952433Z ##[endgroup]
2026-04-12T07:09:15.8955364Z Detected 2 changed files
2026-04-12T07:09:15.8957103Z Results:
2026-04-12T07:09:15.8959641Z ##[group]Filter oci = true
2026-04-12T07:09:15.8961325Z Matching files:
2026-04-12T07:09:15.8962895Z .github/workflows/pi-image.yml [modified]
2026-04-12T07:09:15.8965863Z ##[endgroup]
2026-04-12T07:09:15.8967491Z Changes output set to ["oci"]
2026-04-12T07:09:15.9381849Z ##[group]Run actions/checkout@v5
2026-04-12T07:09:15.9382730Z with:
2026-04-12T07:09:15.9383367Z   fetch-depth: 1
2026-04-12T07:09:15.9384099Z   repository: futuroptimist/sugarkube
2026-04-12T07:09:15.9385205Z   token: ***
2026-04-12T07:09:15.9385851Z   ssh-strict: true
2026-04-12T07:09:15.9386730Z   ssh-user: git
2026-04-12T07:09:15.9387412Z   persist-credentials: true
2026-04-12T07:09:15.9388326Z   clean: true
2026-04-12T07:09:15.9389026Z   sparse-checkout-cone-mode: true
2026-04-12T07:09:15.9389855Z   fetch-tags: false
2026-04-12T07:09:15.9390547Z   show-progress: true
2026-04-12T07:09:15.9391241Z   lfs: false
2026-04-12T07:09:15.9391873Z   submodules: false
2026-04-12T07:09:15.9392563Z   set-safe-directory: true
2026-04-12T07:09:15.9393309Z env:
2026-04-12T07:09:15.9393968Z   FORCE_JAVASCRIPT_ACTIONS_TO_NODE24: true
2026-04-12T07:09:15.9394847Z   CLONE_DSPACE: 
2026-04-12T07:09:15.9395488Z ##[endgroup]
2026-04-12T07:09:16.0216937Z Syncing repository: futuroptimist/sugarkube
2026-04-12T07:09:16.0218864Z ##[group]Getting Git version info
2026-04-12T07:09:16.0219952Z Working directory is '/home/runner/work/sugarkube/sugarkube'
2026-04-12T07:09:16.0221509Z [command]/usr/bin/git version
2026-04-12T07:09:16.0238203Z git version 2.53.0
2026-04-12T07:09:16.0257486Z ##[endgroup]
2026-04-12T07:09:16.0270032Z Temporarily overriding HOME='/home/runner/work/_temp/a7113063-6466-4232-bd9b-91ecafa80828' before making global git config changes
2026-04-12T07:09:16.0272385Z Adding repository directory to the temporary git global config as a safe directory
2026-04-12T07:09:16.0275343Z [command]/usr/bin/git config --global --add safe.directory /home/runner/work/sugarkube/sugarkube
2026-04-12T07:09:16.0301230Z Deleting the contents of '/home/runner/work/sugarkube/sugarkube'
2026-04-12T07:09:16.0304827Z ##[group]Initializing the repository
2026-04-12T07:09:16.0308914Z [command]/usr/bin/git init /home/runner/work/sugarkube/sugarkube
2026-04-12T07:09:16.0393995Z hint: Using 'master' as the name for the initial branch. This default branch name
2026-04-12T07:09:16.0395816Z hint: will change to "main" in Git 3.0. To configure the initial branch name
2026-04-12T07:09:16.0397804Z hint: to use in all of your new repositories, which will suppress this warning,
2026-04-12T07:09:16.0399300Z hint: call:
2026-04-12T07:09:16.0399894Z hint:
2026-04-12T07:09:16.0400727Z hint: 	git config --global init.defaultBranch <name>
2026-04-12T07:09:16.0401789Z hint:
2026-04-12T07:09:16.0403038Z hint: Names commonly chosen instead of 'master' are 'main', 'trunk' and
2026-04-12T07:09:16.0404544Z hint: 'development'. The just-created branch can be renamed via this command:
2026-04-12T07:09:16.0405726Z hint:
2026-04-12T07:09:16.0406358Z hint: 	git branch -m <name>
2026-04-12T07:09:16.0407129Z hint:
2026-04-12T07:09:16.0408432Z hint: Disable this message with "git config set advice.defaultBranchName false"
2026-04-12T07:09:16.0410060Z Initialized empty Git repository in /home/runner/work/sugarkube/sugarkube/.git/
2026-04-12T07:09:16.0412725Z [command]/usr/bin/git remote add origin https://github.com/futuroptimist/sugarkube
2026-04-12T07:09:16.0428790Z ##[endgroup]
2026-04-12T07:09:16.0430002Z ##[group]Disabling automatic garbage collection
2026-04-12T07:09:16.0432114Z [command]/usr/bin/git config --local gc.auto 0
2026-04-12T07:09:16.0454789Z ##[endgroup]
2026-04-12T07:09:16.0455944Z ##[group]Setting up auth
2026-04-12T07:09:16.0459957Z [command]/usr/bin/git config --local --name-only --get-regexp core\.sshCommand
2026-04-12T07:09:16.0484301Z [command]/usr/bin/git submodule foreach --recursive sh -c "git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :"
2026-04-12T07:09:16.0718775Z [command]/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2026-04-12T07:09:16.0744026Z [command]/usr/bin/git submodule foreach --recursive sh -c "git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :"
2026-04-12T07:09:16.0904903Z [command]/usr/bin/git config --local --name-only --get-regexp ^includeIf\.gitdir:
2026-04-12T07:09:16.0927924Z [command]/usr/bin/git submodule foreach --recursive git config --local --show-origin --name-only --get-regexp remote.origin.url
2026-04-12T07:09:16.1089856Z [command]/usr/bin/git config --local http.https://github.com/.extraheader AUTHORIZATION: basic ***
2026-04-12T07:09:16.1115967Z ##[endgroup]
2026-04-12T07:09:16.1117156Z ##[group]Fetching the repository
2026-04-12T07:09:16.1124957Z [command]/usr/bin/git -c protocol.version=2 fetch --no-tags --prune --no-recurse-submodules --depth=1 origin +e2dc882aa7abb8c86337bf4f69f8087d4682cd18:refs/remotes/pull/2162/merge
2026-04-12T07:09:16.5987855Z From https://github.com/futuroptimist/sugarkube
2026-04-12T07:09:16.5989784Z  * [new ref]         e2dc882aa7abb8c86337bf4f69f8087d4682cd18 -> pull/2162/merge
2026-04-12T07:09:16.6009502Z ##[endgroup]
2026-04-12T07:09:16.6010988Z ##[group]Determining the checkout info
2026-04-12T07:09:16.6012283Z ##[endgroup]
2026-04-12T07:09:16.6015557Z [command]/usr/bin/git sparse-checkout disable
2026-04-12T07:09:16.6045238Z [command]/usr/bin/git config --local --unset-all extensions.worktreeConfig
2026-04-12T07:09:16.6066910Z ##[group]Checking out the ref
2026-04-12T07:09:16.6071251Z [command]/usr/bin/git checkout --progress --force refs/remotes/pull/2162/merge
2026-04-12T07:09:16.6537808Z Note: switching to 'refs/remotes/pull/2162/merge'.
2026-04-12T07:09:16.6539199Z 
2026-04-12T07:09:16.6540099Z You are in 'detached HEAD' state. You can look around, make experimental
2026-04-12T07:09:16.6542523Z changes and commit them, and you can discard any commits you make in this
2026-04-12T07:09:16.6545016Z state without impacting any branches by switching back to a branch.
2026-04-12T07:09:16.6546491Z 
2026-04-12T07:09:16.6547521Z If you want to create a new branch to retain commits you create, you may
2026-04-12T07:09:16.6549977Z do so (now or later) by using -c with the switch command. Example:
2026-04-12T07:09:16.6551339Z 
2026-04-12T07:09:16.6551876Z   git switch -c <new-branch-name>
2026-04-12T07:09:16.6552923Z 
2026-04-12T07:09:16.6553451Z Or undo this operation with:
2026-04-12T07:09:16.6553965Z 
2026-04-12T07:09:16.6554246Z   git switch -
2026-04-12T07:09:16.6554655Z 
2026-04-12T07:09:16.6555286Z Turn off this advice by setting config variable advice.detachedHead to false
2026-04-12T07:09:16.6556222Z 
2026-04-12T07:09:16.6557210Z HEAD is now at e2dc882 Merge 54b2299e77e34f055622cefb543b595991beb338 into bf84938928053537acda688b527199ce90b99572
2026-04-12T07:09:16.6561503Z ##[endgroup]
2026-04-12T07:09:16.6575806Z [command]/usr/bin/git log -1 --format=%H
2026-04-12T07:09:16.6594336Z e2dc882aa7abb8c86337bf4f69f8087d4682cd18
2026-04-12T07:09:16.6770144Z ##[group]Run docker/setup-qemu-action@v3
2026-04-12T07:09:16.6771019Z with:
2026-04-12T07:09:16.6771690Z   image: docker.io/tonistiigi/binfmt:latest
2026-04-12T07:09:16.6772558Z   platforms: all
2026-04-12T07:09:16.6773221Z   cache-image: true
2026-04-12T07:09:16.6773851Z env:
2026-04-12T07:09:16.6774503Z   FORCE_JAVASCRIPT_ACTIONS_TO_NODE24: true
2026-04-12T07:09:16.6775359Z   CLONE_DSPACE: 
2026-04-12T07:09:16.6775989Z ##[endgroup]
2026-04-12T07:09:16.8958003Z ##[group]Docker info
2026-04-12T07:09:16.8977922Z (node:2302) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
2026-04-12T07:09:16.8981883Z (Use `node --trace-deprecation ...` to show where the warning was created)
2026-04-12T07:09:16.8995767Z [command]/usr/bin/docker version
2026-04-12T07:09:16.9416115Z Client: Docker Engine - Community
2026-04-12T07:09:16.9418625Z  Version:           28.0.4
2026-04-12T07:09:16.9419088Z  API version:       1.48
2026-04-12T07:09:16.9419620Z  Go version:        go1.23.7
2026-04-12T07:09:16.9420124Z  Git commit:        b8034c0
2026-04-12T07:09:16.9420587Z  Built:             Tue Mar 25 15:07:16 2025
2026-04-12T07:09:16.9421114Z  OS/Arch:           linux/amd64
2026-04-12T07:09:16.9421575Z  Context:           default
2026-04-12T07:09:16.9421835Z 
2026-04-12T07:09:16.9422038Z Server: Docker Engine - Community
2026-04-12T07:09:16.9422481Z  Engine:
2026-04-12T07:09:16.9422834Z   Version:          28.0.4
2026-04-12T07:09:16.9423277Z   API version:      1.48 (minimum version 1.24)
2026-04-12T07:09:16.9424131Z   Go version:       go1.23.7
2026-04-12T07:09:16.9424566Z   Git commit:       6430e49
2026-04-12T07:09:16.9424992Z   Built:            Tue Mar 25 15:07:16 2025
2026-04-12T07:09:16.9425421Z   OS/Arch:          linux/amd64
2026-04-12T07:09:16.9425861Z   Experimental:     false
2026-04-12T07:09:16.9426291Z  containerd:
2026-04-12T07:09:16.9426666Z   Version:          v2.2.2
2026-04-12T07:09:16.9427301Z   GitCommit:        301b2dac98f15c27117da5c8af12118a041a31d9
2026-04-12T07:09:16.9427801Z  runc:
2026-04-12T07:09:16.9428251Z   Version:          1.3.4
2026-04-12T07:09:16.9428617Z   GitCommit:        v1.3.4-0-gd6d73eb8
2026-04-12T07:09:16.9429025Z  docker-init:
2026-04-12T07:09:16.9429361Z   Version:          0.19.0
2026-04-12T07:09:16.9429739Z   GitCommit:        de40ad0
2026-04-12T07:09:16.9461098Z [command]/usr/bin/docker info
2026-04-12T07:09:17.7410747Z Client: Docker Engine - Community
2026-04-12T07:09:17.7411712Z  Version:    28.0.4
2026-04-12T07:09:17.7412107Z  Context:    default
2026-04-12T07:09:17.7412673Z  Debug Mode: false
2026-04-12T07:09:17.7413411Z  Plugins:
2026-04-12T07:09:17.7414437Z   buildx: Docker Buildx (Docker Inc.)
2026-04-12T07:09:17.7415006Z     Version:  v0.33.0
2026-04-12T07:09:17.7415523Z     Path:     /usr/libexec/docker/cli-plugins/docker-buildx
2026-04-12T07:09:17.7416086Z   compose: Docker Compose (Docker Inc.)
2026-04-12T07:09:17.7416526Z     Version:  v2.38.2
2026-04-12T07:09:17.7417012Z     Path:     /usr/libexec/docker/cli-plugins/docker-compose
2026-04-12T07:09:17.7417509Z 
2026-04-12T07:09:17.7417645Z Server:
2026-04-12T07:09:17.7417956Z  Containers: 0
2026-04-12T07:09:17.7418477Z   Running: 0
2026-04-12T07:09:17.7418916Z   Paused: 0
2026-04-12T07:09:17.7419242Z   Stopped: 0
2026-04-12T07:09:17.7419544Z  Images: 6
2026-04-12T07:09:17.7419878Z  Server Version: 28.0.4
2026-04-12T07:09:17.7420312Z  Storage Driver: overlay2
2026-04-12T07:09:17.7420724Z   Backing Filesystem: extfs
2026-04-12T07:09:17.7421140Z   Supports d_type: true
2026-04-12T07:09:17.7421584Z   Using metacopy: false
2026-04-12T07:09:17.7422024Z   Native Overlay Diff: false
2026-04-12T07:09:17.7422453Z   userxattr: false
2026-04-12T07:09:17.7422836Z  Logging Driver: json-file
2026-04-12T07:09:17.7423187Z  Cgroup Driver: systemd
2026-04-12T07:09:17.7423563Z  Cgroup Version: 2
2026-04-12T07:09:17.7423898Z  Plugins:
2026-04-12T07:09:17.7424283Z   Volume: local
2026-04-12T07:09:17.7424738Z   Network: bridge host ipvlan macvlan null overlay
2026-04-12T07:09:17.7426012Z   Log: awslogs fluentd gcplogs gelf journald json-file local splunk syslog
2026-04-12T07:09:17.7426630Z  Swarm: inactive
2026-04-12T07:09:17.7427058Z  Runtimes: io.containerd.runc.v2 runc
2026-04-12T07:09:17.7427575Z  Default Runtime: runc
2026-04-12T07:09:17.7427978Z  Init Binary: docker-init
2026-04-12T07:09:17.7428787Z  containerd version: 301b2dac98f15c27117da5c8af12118a041a31d9
2026-04-12T07:09:17.7429380Z  runc version: v1.3.4-0-gd6d73eb8
2026-04-12T07:09:17.7429832Z  init version: de40ad0
2026-04-12T07:09:17.7430233Z  Security Options:
2026-04-12T07:09:17.7430614Z   apparmor
2026-04-12T07:09:17.7430908Z   seccomp
2026-04-12T07:09:17.7431214Z    Profile: builtin
2026-04-12T07:09:17.7431570Z   cgroupns
2026-04-12T07:09:17.7431910Z  Kernel Version: 6.17.0-1010-azure
2026-04-12T07:09:17.7432396Z  Operating System: Ubuntu 24.04.4 LTS
2026-04-12T07:09:17.7432861Z  OSType: linux
2026-04-12T07:09:17.7433250Z  Architecture: x86_64
2026-04-12T07:09:17.7433586Z  CPUs: 4
2026-04-12T07:09:17.7433926Z  Total Memory: 15.61GiB
2026-04-12T07:09:17.7434300Z  Name: runnervm35a4x
2026-04-12T07:09:17.7434735Z  ID: 87e60688-5316-4409-91d0-e920c95c7873
2026-04-12T07:09:17.7435230Z  Docker Root Dir: /var/lib/docker
2026-04-12T07:09:17.7435652Z  Debug Mode: false
2026-04-12T07:09:17.7436021Z  Username: githubactions
2026-04-12T07:09:17.7436411Z  Experimental: false
2026-04-12T07:09:17.7436793Z  Insecure Registries:
2026-04-12T07:09:17.7442697Z   ::1/128
2026-04-12T07:09:17.7443098Z   127.0.0.0/8
2026-04-12T07:09:17.7443461Z  Live Restore Enabled: false
2026-04-12T07:09:17.7443994Z 
2026-04-12T07:09:17.7444708Z ##[endgroup]
2026-04-12T07:09:17.7445333Z ##[group]Pulling binfmt Docker image
2026-04-12T07:09:17.8324679Z Cache hit for: docker.io--tonistiigi--binfmt-latest-linux-x64
2026-04-12T07:09:17.8423891Z (node:2302) [DEP0169] DeprecationWarning: `url.parse()` behavior is not standardized and prone to errors that have security implications. Use the WHATWG URL API instead. CVEs are not issued for `url.parse()` vulnerabilities.
2026-04-12T07:09:18.3928975Z Received 32159141 of 32159141 (100.0%), 66.2 MBs/sec
2026-04-12T07:09:18.3930592Z Cache Size: ~31 MB (32159141 B)
2026-04-12T07:09:18.3956590Z [command]/usr/bin/tar -xf /home/runner/work/_temp/3d3b441b-9fba-4058-970f-271610bb0e85/cache.tzst -P -C /home/runner/work/sugarkube/sugarkube --use-compress-program unzstd
2026-04-12T07:09:18.5160036Z Cache restored successfully
2026-04-12T07:09:18.5173823Z Restored docker.io--tonistiigi--binfmt-latest-linux-x64 from GitHub Actions cache
2026-04-12T07:09:18.5602346Z Cached to hosted tool cache /opt/hostedtoolcache/docker.io--tonistiigi--binfmt/latest/linux-x64
2026-04-12T07:09:18.6086698Z Copying /opt/hostedtoolcache/docker.io--tonistiigi--binfmt/latest/linux-x64/image.tar to /home/runner/.docker/.cache/images/docker.io--tonistiigi--binfmt/latest/linux-x64/image.tar
2026-04-12T07:09:18.6087644Z Image found from cache in /home/runner/.docker/.cache/images/docker.io--tonistiigi--binfmt/latest/linux-x64/image.tar
2026-04-12T07:09:18.6105069Z [command]/usr/bin/docker load -i /home/runner/.docker/.cache/images/docker.io--tonistiigi--binfmt/latest/linux-x64/image.tar
2026-04-12T07:09:19.1226801Z Loaded image: tonistiigi/binfmt:latest
2026-04-12T07:09:19.1309422Z [command]/usr/bin/docker pull docker.io/tonistiigi/binfmt:latest
2026-04-12T07:09:19.5465650Z latest: Pulling from tonistiigi/binfmt
2026-04-12T07:09:19.6651117Z Digest: sha256:d3b963f787999e6c0219a48dba02978769286ff61a5f4d26245cb6a6e5567ea3
2026-04-12T07:09:19.6656440Z Status: Image is up to date for tonistiigi/binfmt:latest
2026-04-12T07:09:19.6663910Z docker.io/tonistiigi/binfmt:latest
2026-04-12T07:09:19.6686823Z [command]/usr/bin/docker save -o /home/runner/work/_temp/docker-actions-toolkit-Y1z7xf/749e64100d3fa0c09bdbc8a02e4fb9cb3ee64266f7f661c98371040689abacb3.tar docker.io/tonistiigi/binfmt:latest
2026-04-12T07:09:20.0027167Z Copying /home/runner/work/_temp/docker-actions-toolkit-Y1z7xf/749e64100d3fa0c09bdbc8a02e4fb9cb3ee64266f7f661c98371040689abacb3.tar to /home/runner/.docker/.cache/images/docker.io--tonistiigi--binfmt/latest/linux-x64/image.tar
2026-04-12T07:09:20.0654839Z Image cached to /home/runner/.docker/.cache/images/docker.io--tonistiigi--binfmt/latest/linux-x64/image.tar
2026-04-12T07:09:20.0655637Z ##[endgroup]
2026-04-12T07:09:20.0655977Z ##[group]Image info
2026-04-12T07:09:20.0668761Z [command]/usr/bin/docker image inspect docker.io/tonistiigi/binfmt:latest
2026-04-12T07:09:20.0792959Z [
2026-04-12T07:09:20.0793314Z     {
2026-04-12T07:09:20.0793871Z         "Id": "sha256:fb4f8f43716ffaf2802bb32cd60994de149d020f8b1c7e7ddfbdfb43a521ec82",
2026-04-12T07:09:20.0794310Z         "RepoTags": [
2026-04-12T07:09:20.0794523Z             "tonistiigi/binfmt:latest"
2026-04-12T07:09:20.0794753Z         ],
2026-04-12T07:09:20.0794938Z         "RepoDigests": [
2026-04-12T07:09:20.0795360Z             "tonistiigi/binfmt@sha256:d3b963f787999e6c0219a48dba02978769286ff61a5f4d26245cb6a6e5567ea3"
2026-04-12T07:09:20.0795768Z         ],
2026-04-12T07:09:20.0795950Z         "Parent": "",
2026-04-12T07:09:20.0796169Z         "Comment": "buildkit.dockerfile.v0",
2026-04-12T07:09:20.0796436Z         "Created": "2026-02-18T18:09:43.658440389Z",
2026-04-12T07:09:20.0796696Z         "DockerVersion": "",
2026-04-12T07:09:20.0797042Z         "Author": "",
2026-04-12T07:09:20.0797316Z         "Config": {
2026-04-12T07:09:20.0797601Z             "Hostname": "",
2026-04-12T07:09:20.0797918Z             "Domainname": "",
2026-04-12T07:09:20.0798375Z             "User": "",
2026-04-12T07:09:20.0798725Z             "AttachStdin": false,
2026-04-12T07:09:20.0799568Z             "AttachStdout": false,
2026-04-12T07:09:20.0799907Z             "AttachStderr": false,
2026-04-12T07:09:20.0800241Z             "Tty": false,
2026-04-12T07:09:20.0800546Z             "OpenStdin": false,
2026-04-12T07:09:20.0800889Z             "StdinOnce": false,
2026-04-12T07:09:20.0801227Z             "Env": [
2026-04-12T07:09:20.0801731Z                 "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
2026-04-12T07:09:20.0802356Z                 "QEMU_PRESERVE_ARGV0=1"
2026-04-12T07:09:20.0802978Z             ],
2026-04-12T07:09:20.0803472Z             "Cmd": null,
2026-04-12T07:09:20.0803875Z             "Image": "",
2026-04-12T07:09:20.0804360Z             "Volumes": {
2026-04-12T07:09:20.0804785Z                 "/tmp": {}
2026-04-12T07:09:20.0805138Z             },
2026-04-12T07:09:20.0805600Z             "WorkingDir": "/",
2026-04-12T07:09:20.0806039Z             "Entrypoint": [
2026-04-12T07:09:20.0806417Z                 "/usr/bin/binfmt"
2026-04-12T07:09:20.0806916Z             ],
2026-04-12T07:09:20.0807282Z             "OnBuild": null,
2026-04-12T07:09:20.0807697Z             "Labels": null
2026-04-12T07:09:20.0808303Z         },
2026-04-12T07:09:20.0808718Z         "Architecture": "amd64",
2026-04-12T07:09:20.0809200Z         "Os": "linux",
2026-04-12T07:09:20.0809664Z         "Size": 85108073,
2026-04-12T07:09:20.0810121Z         "GraphDriver": {
2026-04-12T07:09:20.0810548Z             "Data": {
2026-04-12T07:09:20.0811497Z                 "LowerDir": "/var/lib/docker/overlay2/e67b783dbb30574f09b741dd5a31ff38bff2b1a09b4f4576b35b65bb02c1da1c/diff",
2026-04-12T07:09:20.0812924Z                 "MergedDir": "/var/lib/docker/overlay2/bcc7d5ead4b1b08a8a843b247e3e822e9f9ecea88738497a4dd48d9b55e53501/merged",
2026-04-12T07:09:20.0814375Z                 "UpperDir": "/var/lib/docker/overlay2/bcc7d5ead4b1b08a8a843b247e3e822e9f9ecea88738497a4dd48d9b55e53501/diff",
2026-04-12T07:09:20.0815803Z                 "WorkDir": "/var/lib/docker/overlay2/bcc7d5ead4b1b08a8a843b247e3e822e9f9ecea88738497a4dd48d9b55e53501/work"
2026-04-12T07:09:20.0816706Z             },
2026-04-12T07:09:20.0817068Z             "Name": "overlay2"
2026-04-12T07:09:20.0817620Z         },
2026-04-12T07:09:20.0817976Z         "RootFS": {
2026-04-12T07:09:20.0818476Z             "Type": "layers",
2026-04-12T07:09:20.0818965Z             "Layers": [
2026-04-12T07:09:20.0819780Z                 "sha256:7c7a3ed8f59c9c48a6885e1b9d666072857891560d4557401a19bd803378d1d4",
2026-04-12T07:09:20.0820735Z                 "sha256:b7dea0aa7b18514e216eebdbded52b5b2c070562d7c88a97f8c3fb0995e4febb"
2026-04-12T07:09:20.0821513Z             ]
2026-04-12T07:09:20.0821894Z         },
2026-04-12T07:09:20.0822216Z         "Metadata": {
2026-04-12T07:09:20.0822865Z             "LastTagTime": "0001-01-01T00:00:00Z"
2026-04-12T07:09:20.0823424Z         }
2026-04-12T07:09:20.0823825Z     }
2026-04-12T07:09:20.0824177Z ]
2026-04-12T07:09:20.0824978Z ##[endgroup]
2026-04-12T07:09:20.0825663Z ##[group]Binfmt version
2026-04-12T07:09:20.0826632Z [command]/usr/bin/docker run --rm --privileged docker.io/tonistiigi/binfmt:latest --version
2026-04-12T07:09:20.5833761Z binfmt/3a63696 qemu/v10.2.1 go/1.25.7
2026-04-12T07:09:20.6344367Z ##[endgroup]
2026-04-12T07:09:20.6347026Z ##[group]Installing QEMU static binaries
2026-04-12T07:09:20.6357006Z [command]/usr/bin/docker run --rm --privileged docker.io/tonistiigi/binfmt:latest --install all
2026-04-12T07:09:20.7305748Z installing: ppc64le OK
2026-04-12T07:09:20.7306106Z installing: mips64le OK
2026-04-12T07:09:20.7306358Z installing: arm64 OK
2026-04-12T07:09:20.7306636Z installing: riscv64 OK
2026-04-12T07:09:20.7306913Z installing: mips64 OK
2026-04-12T07:09:20.7307184Z installing: loong64 OK
2026-04-12T07:09:20.7307447Z installing: arm OK
2026-04-12T07:09:20.7307709Z installing: s390x OK
2026-04-12T07:09:20.7485831Z {
2026-04-12T07:09:20.7486382Z   "supported": [
2026-04-12T07:09:20.7486777Z     "linux/amd64",
2026-04-12T07:09:20.7487129Z     "linux/amd64/v2",
2026-04-12T07:09:20.7487866Z     "linux/amd64/v3",
2026-04-12T07:09:20.7488625Z     "linux/amd64/v4",
2026-04-12T07:09:20.7489004Z     "linux/arm64",
2026-04-12T07:09:20.7489445Z     "linux/riscv64",
2026-04-12T07:09:20.7489863Z     "linux/ppc64le",
2026-04-12T07:09:20.7490233Z     "linux/s390x",
2026-04-12T07:09:20.7490640Z     "linux/386",
2026-04-12T07:09:20.7490997Z     "linux/mips64le",
2026-04-12T07:09:20.7491379Z     "linux/mips64",
2026-04-12T07:09:20.7491818Z     "linux/loong64",
2026-04-12T07:09:20.7492221Z     "linux/arm/v7",
2026-04-12T07:09:20.7492598Z     "linux/arm/v6"
2026-04-12T07:09:20.7493010Z   ],
2026-04-12T07:09:20.7493355Z   "emulators": [
2026-04-12T07:09:20.7493746Z     "llvm-16-runtime.binfmt",
2026-04-12T07:09:20.7494253Z     "llvm-17-runtime.binfmt",
2026-04-12T07:09:20.7494703Z     "llvm-18-runtime.binfmt",
2026-04-12T07:09:20.7495123Z     "python3.12",
2026-04-12T07:09:20.7495555Z     "qemu-aarch64",
2026-04-12T07:09:20.7495891Z     "qemu-arm",
2026-04-12T07:09:20.7497052Z     "qemu-loongarch64",
2026-04-12T07:09:20.7497593Z     "qemu-mips64",
2026-04-12T07:09:20.7498054Z     "qemu-mips64el",
2026-04-12T07:09:20.7498665Z     "qemu-ppc64le",
2026-04-12T07:09:20.7499061Z     "qemu-riscv64",
2026-04-12T07:09:20.7499406Z     "qemu-s390x"
2026-04-12T07:09:20.7499644Z   ]
2026-04-12T07:09:20.7499973Z }
2026-04-12T07:09:20.8014051Z ##[endgroup]
2026-04-12T07:09:20.8014617Z ##[group]Extracting available platforms
2026-04-12T07:09:20.9687902Z linux/amd64,linux/amd64/v2,linux/amd64/v3,linux/amd64/v4,linux/arm64,linux/riscv64,linux/ppc64le,linux/s390x,linux/386,linux/mips64le,linux/mips64,linux/loong64,linux/arm/v7,linux/arm/v6
2026-04-12T07:09:20.9689593Z ##[endgroup]
2026-04-12T07:09:20.9869985Z ##[group]Run docker/setup-buildx-action@v3
2026-04-12T07:09:20.9870356Z with:
2026-04-12T07:09:20.9870658Z   driver: docker-container
2026-04-12T07:09:20.9870918Z   use: true
2026-04-12T07:09:20.9871163Z   keep-state: false
2026-04-12T07:09:20.9871432Z   cache-binary: true
2026-04-12T07:09:20.9871692Z   cleanup: true
2026-04-12T07:09:20.9871966Z   install: false
2026-04-12T07:09:20.9872216Z env:
2026-04-12T07:09:20.9872482Z   FORCE_JAVASCRIPT_ACTIONS_TO_NODE24: true
2026-04-12T07:09:20.9872769Z   CLONE_DSPACE: 
2026-04-12T07:09:20.9873019Z ##[endgroup]
2026-04-12T07:09:21.2139076Z (node:2749) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
2026-04-12T07:09:21.2139839Z (Use `node --trace-deprecation ...` to show where the warning was created)
2026-04-12T07:09:21.2182712Z ##[group]Docker info
2026-04-12T07:09:21.2202059Z [command]/usr/bin/docker version
2026-04-12T07:09:21.2365140Z Client: Docker Engine - Community
2026-04-12T07:09:21.2365982Z  Version:           28.0.4
2026-04-12T07:09:21.2366398Z  API version:       1.48
2026-04-12T07:09:21.2367513Z  Go version:        go1.23.7
2026-04-12T07:09:21.2367963Z  Git commit:        b8034c0
2026-04-12T07:09:21.2368603Z  Built:             Tue Mar 25 15:07:16 2025
2026-04-12T07:09:21.2369231Z  OS/Arch:           linux/amd64
2026-04-12T07:09:21.2369744Z  Context:           default
2026-04-12T07:09:21.2370071Z 
2026-04-12T07:09:21.2370275Z Server: Docker Engine - Community
2026-04-12T07:09:21.2370819Z  Engine:
2026-04-12T07:09:21.2371191Z   Version:          28.0.4
2026-04-12T07:09:21.2371705Z   API version:      1.48 (minimum version 1.24)
2026-04-12T07:09:21.2372329Z   Go version:       go1.23.7
2026-04-12T07:09:21.2372817Z   Git commit:       6430e49
2026-04-12T07:09:21.2373304Z   Built:            Tue Mar 25 15:07:16 2025
2026-04-12T07:09:21.2373872Z   OS/Arch:          linux/amd64
2026-04-12T07:09:21.2374395Z   Experimental:     false
2026-04-12T07:09:21.2374887Z  containerd:
2026-04-12T07:09:21.2375335Z   Version:          v2.2.2
2026-04-12T07:09:21.2375828Z   GitCommit:        301b2dac98f15c27117da5c8af12118a041a31d9
2026-04-12T07:09:21.2376323Z  runc:
2026-04-12T07:09:21.2376715Z   Version:          1.3.4
2026-04-12T07:09:21.2377194Z   GitCommit:        v1.3.4-0-gd6d73eb8
2026-04-12T07:09:21.2377725Z  docker-init:
2026-04-12T07:09:21.2378560Z   Version:          0.19.0
2026-04-12T07:09:21.2379015Z   GitCommit:        de40ad0
2026-04-12T07:09:21.2405295Z [command]/usr/bin/docker info
2026-04-12T07:09:21.2781113Z Client: Docker Engine - Community
2026-04-12T07:09:21.2781693Z  Version:    28.0.4
2026-04-12T07:09:21.2782007Z  Context:    default
2026-04-12T07:09:21.2782290Z  Debug Mode: false
2026-04-12T07:09:21.2782547Z  Plugins:
2026-04-12T07:09:21.2782785Z   buildx: Docker Buildx (Docker Inc.)
2026-04-12T07:09:21.2783110Z     Version:  v0.33.0
2026-04-12T07:09:21.2783471Z     Path:     /usr/libexec/docker/cli-plugins/docker-buildx
2026-04-12T07:09:21.2783856Z   compose: Docker Compose (Docker Inc.)
2026-04-12T07:09:21.2784172Z     Version:  v2.38.2
2026-04-12T07:09:21.2784505Z     Path:     /usr/libexec/docker/cli-plugins/docker-compose
2026-04-12T07:09:21.2784731Z 
2026-04-12T07:09:21.2784885Z Server:
2026-04-12T07:09:21.2785096Z  Containers: 0
2026-04-12T07:09:21.2785361Z   Running: 0
2026-04-12T07:09:21.2785616Z   Paused: 0
2026-04-12T07:09:21.2785827Z   Stopped: 0
2026-04-12T07:09:21.2786064Z  Images: 7
2026-04-12T07:09:21.2786321Z  Server Version: 28.0.4
2026-04-12T07:09:21.2786568Z  Storage Driver: overlay2
2026-04-12T07:09:21.2786869Z   Backing Filesystem: extfs
2026-04-12T07:09:21.2787174Z   Supports d_type: true
2026-04-12T07:09:21.2787411Z   Using metacopy: false
2026-04-12T07:09:21.2787687Z   Native Overlay Diff: false
2026-04-12T07:09:21.2787984Z   userxattr: false
2026-04-12T07:09:21.2788513Z  Logging Driver: json-file
2026-04-12T07:09:21.2788917Z  Cgroup Driver: systemd
2026-04-12T07:09:21.2789394Z  Cgroup Version: 2
2026-04-12T07:09:21.2789835Z  Plugins:
2026-04-12T07:09:21.2790572Z   Volume: local
2026-04-12T07:09:21.2791106Z   Network: bridge host ipvlan macvlan null overlay
2026-04-12T07:09:21.2791902Z   Log: awslogs fluentd gcplogs gelf journald json-file local splunk syslog
2026-04-12T07:09:21.2792573Z  Swarm: inactive
2026-04-12T07:09:21.2793053Z  Runtimes: io.containerd.runc.v2 runc
2026-04-12T07:09:21.2793566Z  Default Runtime: runc
2026-04-12T07:09:21.2794023Z  Init Binary: docker-init
2026-04-12T07:09:21.2794650Z  containerd version: 301b2dac98f15c27117da5c8af12118a041a31d9
2026-04-12T07:09:21.2795363Z  runc version: v1.3.4-0-gd6d73eb8
2026-04-12T07:09:21.2795857Z  init version: de40ad0
2026-04-12T07:09:21.2796273Z  Security Options:
2026-04-12T07:09:21.2796639Z   apparmor
2026-04-12T07:09:21.2796948Z   seccomp
2026-04-12T07:09:21.2797199Z    Profile: builtin
2026-04-12T07:09:21.2797538Z   cgroupns
2026-04-12T07:09:21.2797862Z  Kernel Version: 6.17.0-1010-azure
2026-04-12T07:09:21.2798597Z  Operating System: Ubuntu 24.04.4 LTS
2026-04-12T07:09:21.2798933Z  OSType: linux
2026-04-12T07:09:21.2799178Z  Architecture: x86_64
2026-04-12T07:09:21.2799493Z  CPUs: 4
2026-04-12T07:09:21.2799705Z  Total Memory: 15.61GiB
2026-04-12T07:09:21.2799959Z  Name: runnervm35a4x
2026-04-12T07:09:21.2800262Z  ID: 87e60688-5316-4409-91d0-e920c95c7873
2026-04-12T07:09:21.2800547Z  Docker Root Dir: /var/lib/docker
2026-04-12T07:09:21.2800833Z  Debug Mode: false
2026-04-12T07:09:21.2801137Z  Username: githubactions
2026-04-12T07:09:21.2801381Z  Experimental: false
2026-04-12T07:09:21.2801773Z  Insecure Registries:
2026-04-12T07:09:21.2876654Z   ::1/128
2026-04-12T07:09:21.2876944Z   127.0.0.0/8
2026-04-12T07:09:21.2877257Z  Live Restore Enabled: false
2026-04-12T07:09:21.2877527Z 
2026-04-12T07:09:21.2878077Z ##[endgroup]
2026-04-12T07:09:21.3315674Z ##[group]Buildx version
2026-04-12T07:09:21.3326016Z [command]/usr/bin/docker buildx version
2026-04-12T07:09:21.3765451Z github.com/docker/buildx v0.33.0 f7897eba028583e0071642db3c011e860444f8cf
2026-04-12T07:09:21.3782662Z ##[endgroup]
2026-04-12T07:09:21.3911347Z ##[group]Inspecting default docker context
2026-04-12T07:09:21.4036819Z [
2026-04-12T07:09:21.4037124Z   {
2026-04-12T07:09:21.4037388Z     "Name": "default",
2026-04-12T07:09:21.4037746Z     "Metadata": {},
2026-04-12T07:09:21.4038062Z     "Endpoints": {
2026-04-12T07:09:21.4038490Z       "docker": {
2026-04-12T07:09:21.4039187Z         "Host": "unix:///var/run/docker.sock",
2026-04-12T07:09:21.4039607Z         "SkipTLSVerify": false
2026-04-12T07:09:21.4039947Z       }
2026-04-12T07:09:21.4040177Z     },
2026-04-12T07:09:21.4040428Z     "TLSMaterial": {},
2026-04-12T07:09:21.4040712Z     "Storage": {
2026-04-12T07:09:21.4041017Z       "MetadataPath": "<IN MEMORY>",
2026-04-12T07:09:21.4041413Z       "TLSPath": "<IN MEMORY>"
2026-04-12T07:09:21.4041741Z     }
2026-04-12T07:09:21.4041989Z   }
2026-04-12T07:09:21.4042250Z ]
2026-04-12T07:09:21.4042882Z ##[endgroup]
2026-04-12T07:09:21.4043446Z ##[group]Creating a new builder instance
2026-04-12T07:09:21.5173830Z [command]/usr/bin/docker buildx create --name builder-c1b5c4b1-88b3-45cb-a75a-0f26f86e0488 --driver docker-container --buildkitd-flags --allow-insecure-entitlement security.insecure --allow-insecure-entitlement network.host --use
2026-04-12T07:09:21.5727429Z builder-c1b5c4b1-88b3-45cb-a75a-0f26f86e0488
2026-04-12T07:09:21.5761438Z ##[endgroup]
2026-04-12T07:09:21.5767640Z ##[group]Booting builder
2026-04-12T07:09:21.5791751Z [command]/usr/bin/docker buildx inspect --bootstrap --builder builder-c1b5c4b1-88b3-45cb-a75a-0f26f86e0488
2026-04-12T07:09:21.6264415Z #1 [internal] booting buildkit
2026-04-12T07:09:21.7770004Z #1 pulling image moby/buildkit:buildx-stable-1
2026-04-12T07:09:24.6368244Z #1 pulling image moby/buildkit:buildx-stable-1 3.0s done
2026-04-12T07:09:24.7871234Z #1 creating container buildx_buildkit_builder-c1b5c4b1-88b3-45cb-a75a-0f26f86e04880
2026-04-12T07:09:24.9371813Z #1 creating container buildx_buildkit_builder-c1b5c4b1-88b3-45cb-a75a-0f26f86e04880 0.3s done
2026-04-12T07:09:24.9386582Z #1 DONE 3.3s
2026-04-12T07:09:24.9677882Z Name:          builder-c1b5c4b1-88b3-45cb-a75a-0f26f86e0488
2026-04-12T07:09:24.9678668Z Driver:        docker-container
2026-04-12T07:09:24.9679069Z Last Activity: 2026-04-12 07:09:21 +0000 UTC
2026-04-12T07:09:24.9679404Z 
2026-04-12T07:09:24.9679518Z Nodes:
2026-04-12T07:09:24.9679931Z Name:                  builder-c1b5c4b1-88b3-45cb-a75a-0f26f86e04880
2026-04-12T07:09:24.9682804Z Endpoint:              unix:///var/run/docker.sock
2026-04-12T07:09:24.9683083Z Status:                running
2026-04-12T07:09:24.9683550Z BuildKit daemon flags: --allow-insecure-entitlement security.insecure --allow-insecure-entitlement network.host
2026-04-12T07:09:24.9684042Z BuildKit version:      v0.29.0
2026-04-12T07:09:24.9684780Z Platforms:             linux/amd64, linux/amd64/v2, linux/amd64/v3, linux/amd64/v4, linux/arm64, linux/riscv64, linux/ppc64le, linux/s390x, linux/386, linux/mips64le, linux/mips64, linux/loong64, linux/arm/v7, linux/arm/v6
2026-04-12T07:09:24.9685486Z Labels:
2026-04-12T07:09:24.9685740Z  org.mobyproject.buildkit.worker.executor:         oci
2026-04-12T07:09:24.9686133Z  org.mobyproject.buildkit.worker.hostname:         3eab1b195b2e
2026-04-12T07:09:24.9686513Z  org.mobyproject.buildkit.worker.network:          host
2026-04-12T07:09:24.9686904Z  org.mobyproject.buildkit.worker.oci.process-mode: sandbox
2026-04-12T07:09:24.9687325Z  org.mobyproject.buildkit.worker.selinux.enabled:  false
2026-04-12T07:09:24.9687712Z  org.mobyproject.buildkit.worker.snapshotter:      overlayfs
2026-04-12T07:09:24.9688018Z GC Policy rule#0:
2026-04-12T07:09:24.9688391Z  All:            false
2026-04-12T07:09:24.9688789Z  Filters:        type==source.local,type==exec.cachemount,type==source.git.checkout
2026-04-12T07:09:24.9689148Z  Keep Duration:  48h0m0s
2026-04-12T07:09:24.9689353Z  Max Used Space: 488.3MiB
2026-04-12T07:09:24.9689573Z GC Policy rule#1:
2026-04-12T07:09:24.9689753Z  All:            false
2026-04-12T07:09:24.9689943Z  Keep Duration:  1440h0m0s
2026-04-12T07:09:24.9690152Z  Reserved Space: 9.313GiB
2026-04-12T07:09:24.9690353Z  Max Used Space: 93.13GiB
2026-04-12T07:09:24.9690553Z  Min Free Space: 27.01GiB
2026-04-12T07:09:24.9690734Z GC Policy rule#2:
2026-04-12T07:09:24.9690908Z  All:            false
2026-04-12T07:09:24.9691184Z  Reserved Space: 9.313GiB
2026-04-12T07:09:24.9691527Z  Max Used Space: 93.13GiB
2026-04-12T07:09:24.9691978Z  Min Free Space: 27.01GiB
2026-04-12T07:09:24.9692203Z GC Policy rule#3:
2026-04-12T07:09:24.9692476Z  All:            true
2026-04-12T07:09:24.9692695Z  Reserved Space: 9.313GiB
2026-04-12T07:09:24.9692891Z  Max Used Space: 93.13GiB
2026-04-12T07:09:24.9693076Z  Min Free Space: 27.01GiB
2026-04-12T07:09:24.9713744Z ##[endgroup]
2026-04-12T07:09:25.0512894Z ##[group]Inspect builder
2026-04-12T07:09:25.0514855Z {
2026-04-12T07:09:25.0515224Z   "nodes": [
2026-04-12T07:09:25.0515478Z     {
2026-04-12T07:09:25.0516224Z       "name": "builder-c1b5c4b1-88b3-45cb-a75a-0f26f86e04880",
2026-04-12T07:09:25.0516995Z       "endpoint": "unix:///var/run/docker.sock",
2026-04-12T07:09:25.0517336Z       "status": "running",
2026-04-12T07:09:25.0517972Z       "buildkitd-flags": "--allow-insecure-entitlement security.insecure --allow-insecure-entitlement network.host",
2026-04-12T07:09:25.0518846Z       "buildkit": "v0.29.0",
2026-04-12T07:09:25.0519535Z       "platforms": "linux/amd64,linux/amd64/v2,linux/amd64/v3,linux/amd64/v4,linux/arm64,linux/riscv64,linux/ppc64le,linux/s390x,linux/386,linux/mips64le,linux/mips64,linux/loong64,linux/arm/v7,linux/arm/v6",
2026-04-12T07:09:25.0520241Z       "features": {
2026-04-12T07:09:25.0520525Z         "Automatically load images to the Docker Engine image store": true,
2026-04-12T07:09:25.0520862Z         "Cache export": true,
2026-04-12T07:09:25.0521074Z         "Direct push": true,
2026-04-12T07:09:25.0521292Z         "Docker exporter": true,
2026-04-12T07:09:25.0521519Z         "Multi-platform build": true,
2026-04-12T07:09:25.0521745Z         "OCI exporter": true
2026-04-12T07:09:25.0521953Z       },
2026-04-12T07:09:25.0522316Z       "labels": {
2026-04-12T07:09:25.0523213Z         "org.mobyproject.buildkit.worker.executor": "oci",
2026-04-12T07:09:25.0523758Z         "org.mobyproject.buildkit.worker.hostname": "3eab1b195b2e",
2026-04-12T07:09:25.0524225Z         "org.mobyproject.buildkit.worker.network": "host",
2026-04-12T07:09:25.0524743Z         "org.mobyproject.buildkit.worker.oci.process-mode": "sandbox",
2026-04-12T07:09:25.0525318Z         "org.mobyproject.buildkit.worker.selinux.enabled": "false",
2026-04-12T07:09:25.0525950Z         "org.mobyproject.buildkit.worker.snapshotter": "overlayfs"
2026-04-12T07:09:25.0526474Z       },
2026-04-12T07:09:25.0526795Z       "gcPolicy": [
2026-04-12T07:09:25.0527115Z         {
2026-04-12T07:09:25.0527432Z           "all": false,
2026-04-12T07:09:25.0527772Z           "filter": [
2026-04-12T07:09:25.0528220Z             "type==source.local",
2026-04-12T07:09:25.0529035Z             "type==exec.cachemount",
2026-04-12T07:09:25.0529472Z             "type==source.git.checkout"
2026-04-12T07:09:25.0529851Z           ],
2026-04-12T07:09:25.0530213Z           "keepDuration": "48h0m0s",
2026-04-12T07:09:25.0530625Z           "maxUsedSpace": "488.3MiB"
2026-04-12T07:09:25.0531038Z         },
2026-04-12T07:09:25.0531328Z         {
2026-04-12T07:09:25.0531564Z           "all": false,
2026-04-12T07:09:25.0531888Z           "keepDuration": "1440h0m0s",
2026-04-12T07:09:25.0532309Z           "reservedSpace": "9.313GiB",
2026-04-12T07:09:25.0532713Z           "maxUsedSpace": "93.13GiB",
2026-04-12T07:09:25.0533142Z           "minFreeSpace": "27.01GiB"
2026-04-12T07:09:25.0534811Z         },
2026-04-12T07:09:25.0535482Z         {
2026-04-12T07:09:25.0535883Z           "all": false,
2026-04-12T07:09:25.0536230Z           "reservedSpace": "9.313GiB",
2026-04-12T07:09:25.0536653Z           "maxUsedSpace": "93.13GiB",
2026-04-12T07:09:25.0537112Z           "minFreeSpace": "27.01GiB"
2026-04-12T07:09:25.0537545Z         },
2026-04-12T07:09:25.0537840Z         {
2026-04-12T07:09:25.0538205Z           "all": true,
2026-04-12T07:09:25.0538571Z           "reservedSpace": "9.313GiB",
2026-04-12T07:09:25.0538834Z           "maxUsedSpace": "93.13GiB",
2026-04-12T07:09:25.0539116Z           "minFreeSpace": "27.01GiB"
2026-04-12T07:09:25.0539434Z         }
2026-04-12T07:09:25.0539593Z       ]
2026-04-12T07:09:25.0539863Z     }
2026-04-12T07:09:25.0540114Z   ],
2026-04-12T07:09:25.0540837Z   "name": "builder-c1b5c4b1-88b3-45cb-a75a-0f26f86e0488",
2026-04-12T07:09:25.0541651Z   "driver": "docker-container",
2026-04-12T07:09:25.0541952Z   "lastActivity": "2026-04-12T07:09:21.000Z"
2026-04-12T07:09:25.0542469Z }
2026-04-12T07:09:25.0542897Z ##[endgroup]
2026-04-12T07:09:25.0543478Z ##[group]BuildKit version
2026-04-12T07:09:25.0543787Z builder-c1b5c4b1-88b3-45cb-a75a-0f26f86e04880: v0.29.0
2026-04-12T07:09:25.0544390Z ##[endgroup]
2026-04-12T07:09:25.0643196Z ##[group]Run set -euo pipefail
2026-04-12T07:09:25.0643573Z [36;1mset -euo pipefail[0m
2026-04-12T07:09:25.0643898Z [36;1mworkspace_repo="${GITHUB_WORKSPACE}/dspace"[0m
2026-04-12T07:09:25.0644259Z [36;1mif [ -d "${workspace_repo}" ]; then[0m
2026-04-12T07:09:25.0644631Z [36;1m  echo "Found existing dspace checkout at ${workspace_repo}" >&2[0m
2026-04-12T07:09:25.0645101Z [36;1m  echo "path=${workspace_repo}" >>"${GITHUB_OUTPUT}"[0m
2026-04-12T07:09:25.0645403Z [36;1m  exit 0[0m
2026-04-12T07:09:25.0645587Z [36;1mfi[0m
2026-04-12T07:09:25.0645759Z [36;1m[0m
2026-04-12T07:09:25.0645947Z [36;1mtarget_dir="${workspace_repo}"[0m
2026-04-12T07:09:25.0646305Z [36;1mif [ -z "${CLONE_DSPACE_INPUT}" ] || [ "${CLONE_DSPACE_INPUT}" != "true" ]; then[0m
2026-04-12T07:09:25.0646674Z [36;1m  target_dir="$(mktemp -d)"[0m
2026-04-12T07:09:25.0646904Z [36;1mfi[0m
2026-04-12T07:09:25.0647062Z [36;1m[0m
2026-04-12T07:09:25.0647487Z [36;1mgit clone --depth=1 --branch v3 https://github.com/democratizedspace/dspace.git "${target_dir}"[0m
2026-04-12T07:09:25.0647974Z [36;1mecho "path=${target_dir}" >>"${GITHUB_OUTPUT}"[0m
2026-04-12T07:09:25.0665852Z shell: /usr/bin/bash -e {0}
2026-04-12T07:09:25.0666085Z env:
2026-04-12T07:09:25.0666286Z   FORCE_JAVASCRIPT_ACTIONS_TO_NODE24: true
2026-04-12T07:09:25.0666534Z   CLONE_DSPACE: 
2026-04-12T07:09:25.0666730Z   CLONE_DSPACE_INPUT: 
2026-04-12T07:09:25.0666925Z ##[endgroup]
2026-04-12T07:09:25.0739405Z Cloning into '/tmp/tmp.cmg1LoasYq'...
2026-04-12T07:09:25.2309542Z fatal: Remote branch v3 not found in upstream origin
2026-04-12T07:09:25.2350058Z ##[error]Process completed with exit code 128.
2026-04-12T07:09:25.2432227Z Post job cleanup.
2026-04-12T07:09:25.4868543Z ##[group]Removing builder
2026-04-12T07:09:25.4875024Z (node:3281) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
2026-04-12T07:09:25.4876328Z (Use `node --trace-deprecation ...` to show where the warning was created)
2026-04-12T07:09:25.5690703Z [command]/usr/bin/docker buildx rm builder-c1b5c4b1-88b3-45cb-a75a-0f26f86e0488
2026-04-12T07:09:25.6763567Z builder-c1b5c4b1-88b3-45cb-a75a-0f26f86e0488 removed
2026-04-12T07:09:25.6784460Z ##[endgroup]
2026-04-12T07:09:25.6784921Z ##[group]Cleaning up certificates
2026-04-12T07:09:25.6786198Z ##[endgroup]
2026-04-12T07:09:25.6787916Z ##[group]Post cache
2026-04-12T07:09:25.6789322Z State not set
2026-04-12T07:09:25.6789716Z ##[endgroup]
2026-04-12T07:09:25.6885222Z Post job cleanup.
2026-04-12T07:09:25.9126798Z ##[group]Post cache
2026-04-12T07:09:25.9129778Z Caching docker.io--tonistiigi--binfmt-latest-linux-x64 to GitHub Actions cache
2026-04-12T07:09:25.9149076Z (node:3402) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
2026-04-12T07:09:25.9150047Z (Use `node --trace-deprecation ...` to show where the warning was created)
2026-04-12T07:09:25.9283721Z [command]/usr/bin/tar --posix -cf cache.tzst --exclude cache.tzst -P -C /home/runner/work/sugarkube/sugarkube --files-from manifest.txt --use-compress-program zstdmt
2026-04-12T07:09:26.2366835Z Failed to save: Unable to reserve cache with key docker.io--tonistiigi--binfmt-latest-linux-x64, another job may be creating this cache.
2026-04-12T07:09:26.2386496Z ##[endgroup]
2026-04-12T07:09:26.2499461Z Post job cleanup.
2026-04-12T07:09:26.3239091Z [command]/usr/bin/git version
2026-04-12T07:09:26.3270746Z git version 2.53.0
2026-04-12T07:09:26.3302568Z Temporarily overriding HOME='/home/runner/work/_temp/392e411e-0420-44e2-af43-8c22d1d5d353' before making global git config changes
2026-04-12T07:09:26.3303681Z Adding repository directory to the temporary git global config as a safe directory
2026-04-12T07:09:26.3307934Z [command]/usr/bin/git config --global --add safe.directory /home/runner/work/sugarkube/sugarkube
2026-04-12T07:09:26.3338471Z [command]/usr/bin/git config --local --name-only --get-regexp core\.sshCommand
2026-04-12T07:09:26.3364285Z [command]/usr/bin/git submodule foreach --recursive sh -c "git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :"
2026-04-12T07:09:26.3534913Z [command]/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2026-04-12T07:09:26.3552952Z http.https://github.com/.extraheader
2026-04-12T07:09:26.3561150Z [command]/usr/bin/git config --local --unset-all http.https://github.com/.extraheader
2026-04-12T07:09:26.3584607Z [command]/usr/bin/git submodule foreach --recursive sh -c "git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :"
2026-04-12T07:09:26.3749359Z [command]/usr/bin/git config --local --name-only --get-regexp ^includeIf\.gitdir:
2026-04-12T07:09:26.3772542Z [command]/usr/bin/git submodule foreach --recursive git config --local --show-origin --name-only --get-regexp remote.origin.url
2026-04-12T07:09:26.4042525Z Cleaning up orphan processes
2026-04-12T07:09:26.4314770Z ##[warning]Node.js 20 is deprecated. The following actions target Node.js 20 but are being forced to run on Node.js 24: docker/setup-buildx-action@v3, docker/setup-qemu-action@v3, dorny/paths-filter@de90cc6fb38fc0963ad72b210f1f284cd68cea36. For more information see: https://github.blog/changelog/2025-09-19-deprecation-of-node-20-on-github-actions-runners/
```

new codex task, not a r/e/v/i/e/w task

--- a/.github/workflows/pi-image.yml
+++ b/.github/workflows/pi-image.yml
@@ -149,6 +149,10 @@ jobs:
               --platform "${platform}" \
               --tag "${tag}" \
               --build-arg DSPACE_VERSION="ci-${GITHUB_SHA}" \
+              --build-arg GITHUB_SHA="${GITHUB_SHA}" \
+              --build-arg VITE_GIT_SHA="${GITHUB_SHA}" \
+              --build-arg GIT_SHA="${GITHUB_SHA}" \
+              --build-arg DSPACE_GIT_SHA="${GITHUB_SHA}" \
               --file "${DSPACE_DIR}/Dockerfile" \
               "${DSPACE_DIR}" \
               --load

--- a/.github/workflows/pi-image.yml
+++ b/.github/workflows/pi-image.yml
@@ -13,7 +13,7 @@ on:
         type: boolean
         default: false
       clone_dspace:
-        description: "Clone democratizedspace/dspace v3 repo into image"
+        description: "Clone democratizedspace/dspace v3.0.0 tag into image"
         type: boolean
         default: false
   # Also run the lightweight unit tests automatically when relevant bits change
@@ -133,7 +133,7 @@ jobs:
             target_dir="$(mktemp -d)"
           fi
 
-          git clone --depth=1 --branch v3 https://github.com/democratizedspace/dspace.git "${target_dir}"
+          git clone --depth=1 --branch v3.0.0 https://github.com/democratizedspace/dspace.git "${target_dir}"
           echo "path=${target_dir}" >>"${GITHUB_OUTPUT}"
 
       - name: Build dspace OCI images (amd64 + arm64)

--- a/tests/test_pi_image_tooling.py
+++ b/tests/test_pi_image_tooling.py
@@ -666,6 +666,14 @@ def test_pi_image_workflow_unit_job_has_fork_guardrails():
     assert "Checkout pull request head" not in unit_job_content
 
 
+def test_pi_image_workflow_clones_existing_dspace_ref():
+    workflow_path = Path(".github/workflows/pi-image.yml")
+    content = workflow_path.read_text()
+
+    assert "--branch v3.0.0" in content
+    assert "--branch v3 https://github.com/democratizedspace/dspace.git" not in content
+
+
 def test_pi_image_workflow_pull_request_paths_include_oci_signals():
     workflow_path = Path(".github/workflows/pi-image.yml")
     content = workflow_path.read_text()

--- a/tests/test_pi_image_tooling.py
+++ b/tests/test_pi_image_tooling.py
@@ -674,6 +674,15 @@ def test_pi_image_workflow_clones_existing_dspace_ref():
     assert "--branch v3 https://github.com/democratizedspace/dspace.git" not in content
 
 
+def test_pi_image_workflow_passes_git_sha_build_args_to_dspace_build():
+    workflow_path = Path(".github/workflows/pi-image.yml")
+    content = workflow_path.read_text()
+
+    assert '--build-arg VITE_GIT_SHA="${GITHUB_SHA}"' in content
+    assert '--build-arg GIT_SHA="${GITHUB_SHA}"' in content
+    assert '--build-arg DSPACE_GIT_SHA="${GITHUB_SHA}"' in content
+
+
 def test_pi_image_workflow_pull_request_paths_include_oci_signals():
     workflow_path = Path(".github/workflows/pi-image.yml")
     content = workflow_path.read_text()

